app/views/spree/shared/_footer.html.erb

```diff
@@ -86,7 +86,107 @@
 <%# footer is replaced overall %>
 <% cache spree_menu_cache_key('footer') do %>
   <footer id="footer" class="footer-wrapper" >
-    <%= render partial: 'spree/shared/upper_footer' %>
```

Comment:
@hamzanaveed9 , why we removed this partial? Any specific reason?

Reply:
resolved it sir
just moved it here to make it dynamic